### PR TITLE
Update the account-name syntax for CreateAccount command

### PR DESCRIPTION
**Pull request by mitsutaka-takeda**
See this PR for the detail.

https://github.com/hyperledger/iroha/pull/694

### CODE REVIEW

**Comment by l4l** [comment]
imo note of rfc number is enough, no need of full description, what do you think @neewy ?

**Comment by neewy** [comment]
I think this is a nice suggestion to add, because it is more convenient to keep short description :)

**Comment by mitsutaka-takeda** [comment]
@neewy Let me clarify before I make a change: "I think this is a nice suggestion to add". You mean keep just rfc number?

**Comment by neewy** [comment]
@mitsutaka-takeda I meant your contribution, so no problems with that :) no need to change it

File: source/includes/_commands.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ message CreateAccount {
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Account name | domain-unique name for account | `[a-z]{1,7}`
+Account name | domain-unique name for account | A string in domain-name syntax defined in RFC1035. An account name is a list of lables separeted by a period `.`. A lable is a sequence of characters in `[a-zA-Z-]`. The length of a lable must not exceed 63 characters.
```
Comment (neewy) [inline source/includes/_commands.md]: please, change to `separated` instead of `separeted`

**Comment by neewy** [comment]
Thank you for your contribution. I had to merge it in my code rather than merge it into docs branch. In future, there will be a contribution guide available.

Happy holidays! 🎄🎅 🎁 🕎

**Pull request closed without merging**